PiwikTracker/Dispatcher.swift

```diff
@@ -1,9 +1,7 @@
 import Foundation
 
-internal protocol Dispatcher {    
+public protocol Dispatcher {
     var userAgent: String? { get }
-    
-    func send(event: Event, success: ()->(), failure: (_ shouldContinue: Bool)->());
-    
-    func send(events: [Event], success: ()->(), failure: (_ shouldContinue: Bool)->());
+        
+    func send(events: [Event], success: ()->(), failure: ()->());
```

Comment:
After an unsuccessful transmission we aren't allowed to send the other events anyways. So the shouldContinue parameter is nonsense.